[2025-03-25T15:12:45.723+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: frequent_flyers.create_table scheduled__2025-03-24T02:00:00+00:00 [queued]>
[2025-03-25T15:12:45.743+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: frequent_flyers.create_table scheduled__2025-03-24T02:00:00+00:00 [queued]>
[2025-03-25T15:12:45.744+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-25T15:12:45.765+0000] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): create_table> on 2025-03-24 02:00:00+00:00
[2025-03-25T15:12:45.770+0000] {standard_task_runner.py:57} INFO - Started process 219 to run task
[2025-03-25T15:12:45.780+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'frequent_flyers', 'create_table', 'scheduled__2025-03-24T02:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/frequent_flyers.py', '--cfg-path', '/tmp/tmp43p72ab9']
[2025-03-25T15:12:45.787+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask create_table
[2025-03-25T15:12:45.813+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-25T15:12:45.919+0000] {task_command.py:415} INFO - Running <TaskInstance: frequent_flyers.create_table scheduled__2025-03-24T02:00:00+00:00 [running]> on host f489f841bffe
[2025-03-25T15:12:46.048+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='frequent_flyers' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-24T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-24T02:00:00+00:00'
[2025-03-25T15:12:46.053+0000] {sql.py:274} INFO - Executing: 
CREATE TABLE IF NOT EXISTS presentation.frequent_flyers (
    created_at TIMESTAMP DEFAULT NOW(),
    passenger_id VARCHAR(20),
    passenger_name TEXT,
    flights_number INTEGER,
    purchase_sum NUMERIC(10, 2),
    home_airport CHAR(3),
    customer_group VARCHAR(5),
    CONSTRAINT pk_frequent_flyers PRIMARY KEY (passenger_id)
);

DELETE FROM presentation.frequent_flyers
WHERE created_at < DATE_TRUNC('day', NOW()) - INTERVAL '1 day';
[2025-03-25T15:12:46.069+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-25T15:12:46.086+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-25T15:12:46.097+0000] {sql.py:418} INFO - Running statement: 
CREATE TABLE IF NOT EXISTS presentation.frequent_flyers (
    created_at TIMESTAMP DEFAULT NOW(),
    passenger_id VARCHAR(20),
    passenger_name TEXT,
    flights_number INTEGER,
    purchase_sum NUMERIC(10, 2),
    home_airport CHAR(3),
    customer_group VARCHAR(5),
    CONSTRAINT pk_frequent_flyers PRIMARY KEY (passenger_id)
);

DELETE FROM presentation.frequent_flyers
WHERE created_at < DATE_TRUNC('day', NOW()) - INTERVAL '1 day';, parameters: None
[2025-03-25T15:12:46.163+0000] {sql.py:427} INFO - Rows affected: 0
[2025-03-25T15:12:46.183+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=frequent_flyers, task_id=create_table, execution_date=20250324T020000, start_date=20250325T151245, end_date=20250325T151246
[2025-03-25T15:12:46.202+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-25T15:12:46.239+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
